core/src/test/java/pt/webdetails/cpf/repository/VfsRepositoryTest.java

```diff
@@ -62,14 +68,17 @@ public void testFolderCreation() throws IOException {
     assertTrue( doNothing );
   }
 
-  public void testResourceExists() throws IOException {
+  @Test
+  public void testResourceExists() throws Exception {
     repository.createFolder( "testFolderCreation" );
     boolean exists = repository.resourceExists( "testFolderCreation" );
+    assertTrue( exists );
     boolean notExists = repository.resourceExists( "IAmNotAFolder!" );
```

Comment:
This variable has a ambiguous name because has the name notExists but the value is false instead of true.

Reply:
Ok, I can remove the variables and make this simpler.